articles/language/type-model.md

```diff
@@ -338,30 +338,31 @@ with the same result type and an input type that is compatible with `'A`.
 That is, given the following definitions:
 
 ```qsharp
-operation Invertible (qs : Qubit[]) : Unit 
+operation Invert(qs : Qubit[]) : Unit 
 is Adj {...} 
-operation Unitary (qs : Qubit[]) : Unit 
+
+operation Unitary(qs : Qubit[]) : Unit 
 is Adj + Ctl {...} 
 
-function ConjugateInvertibleWith (
-   inner: (Qubit[] => Unit is Adj),
-   outer : (Qubit[] => Unit is Adj))
+function ConjugateInvertWith(
+    inner : (Qubit[] => Unit is Adj),
+    outer : (Qubit[] => Unit is Adj))
 : (Qubit[] => Unit is Adj) {...}
 
-function ConjugateUnitaryWith (
-   inner: (Qubit[] => Unit is Adj + Ctl),
-   outer : (Qubit[] => Unit is Adj))
+function ConjugateUnitaryWith(
+    inner : (Qubit[] => Unit is Adj + Ctl),
+    outer : (Qubit[] => Unit is Adj))
 : (Qubit[] => Unit is Adj + Ctl) {...}
 ```
 
 the following are true:
 
-- The operation `ConjugateInvertibleWith` may be invoked with an `inner`
-  argument of either `Invertible` or `Unitary`.
+- The operation `ConjugateInvertWith` may be invoked with an `inner`
+  argument of either `Invert` or `Unitary`.
```

Comment:
Here there is an inconsistency. Is `ConjugateInvertWith` a function or an operation?

Reply:
`ConjugateInvertWith` is a function, and `Invert` is an operation. The former takes operations to other operations, and the latter takes `Qubit[]` and returns `Unit`